src/collectors/portcheck/portcheck.py

```diff
@@ -0,0 +1,108 @@
+"""
+The PortCheckCollector checks ports listed in config file.
+
+##### Dependencies
+
+* psutil
+
+Example config file PortCheckCollector.conf
+
+```
+enabled = True
+ttl = 120
+
+[port]
+[[echo]]
+number = 8080
```

Comment:
Does this only support checking a local port, not a remote port?

Reply:
right, only local tcp port for now.

Reply:
The Nagios check referenced in the requirements allows for remote ports to be checked, you might want to confirm if this is a requirement or not: https://exchange.nagios.org/directory/Plugins/Network-Protocols/*-TCP-and-UDP-(Generic)/check_port-2Epl/details

Reply:
confirmed this is not a requirement